### ISSUE TITLE
Quick Run

### ISSUE THREAD

**Issue by FLO-2DKaren** [FLO-2DSoftware/qgis-flo-2d-plugin#1020]
Use a quick run but modify it if there is a storm drain.

**Comment by rpachaly** [comment]
The quick run button ask the user to select the directory where FLO-2D will run, then the components and runs FLO-2D Pro. It saves some clicks for users that don't like to check the *.DAT files. 

Maybe in the future we could remove the components dialog, but for now I wanted to keep it. 

Here is the final layout for the quick run.

![Image](https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/assets/39889306/5ea53bb7-901c-435d-bdb4-caa778794805)

I changed the icons. I kept the FLO with the globe icon for the quick run because it runs directly with the exported data (more stuff associated with the button) and the FLO icon for Run FLO-2D Pro because it is just runs the model. See if you like it, it is really easy to change these stuff. 

It is on this [branch](https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/tree/quick_run).
